cli/.js/src/main/scala/aqua/run/FuncCaller.scala

```diff
@@ -100,7 +100,7 @@ object FuncCaller {
   }
 
   val TimeoutErrorMessage =
-    "Function execution failed by timeout. You can increase timeout with '--timeout' option in milliseconds or check if your code can hang while executing."
+    "Function execution failed by timeout. You can increase the timeout with '--timeout' option in milliseconds or check if your code can hang while executing."
```

Comment:
timeout is a concept, there's no "that timeout" or "another timeout" or "a timeout".

Reply:
I think here it's better to have 'the' article because we reference something from the previous sentence. It's a particular timeout that we are talking about, not any timeout